packages/server/src/login/login.controller.ts

```diff
@@ -165,4 +165,18 @@ export class LoginController {
       .cookie('auth_token', authToken, { httpOnly: true, secure: isSecure })
       .redirect(302, '/');
   }
+
+  @Get('/logout')
+  async logout(@Res() res: Response): Promise<void> {
+    const isSecure = this.configService
+      .get<string>('DOMAIN')
+      .startsWith('https://');
+    res
+      .cookie('auth_token', '', {
+        httpOnly: true,
+        secure: isSecure,
+        expires: new Date(Date.now() - 86400000),
+      })
+      .redirect(302, '/');
```

Comment:
would it make sense to send them to `/login` instead?

Reply:
wouldn't this do that by default? i'm just letting the app's default behaviour decide